LayoutTests/fast/css-grid-layout/text-resizing-overflow-expected.txt

```diff
@@ -0,0 +1,13 @@
+layer at (0,0) size 800x600
```

Comment:
If the issue is a missing layout, can we test this by having a ref that performs a layout, and a test that does not perform a layout ?

Reply:
though it might be possible to make a more straightforward ref that isn't a render tree dump

Reply:
yeah, or just come up with something that looks exactly like the correct rendering. Render tree dumps have huge maintenance burden and for that reason we stopped producing them for a while now.

Reply:
Switched to HTML expected.